Add KYMA_VERSION readme (#1436)
kyma-incubator/compass: installation/resources/readme.md

# Compass Resources

> ## Installation CRs

Reviewer: The table lists files containing different CRs and not really CR names themselves. 
Maybe it would be useful to add a sentence describing the table above it? Sth like in my suggestion (feel free to modify it to sth more appropriate). 

```suggestion
## Installation CRs

The following table lists all the Installation custom resource files for Compass:
```

| File                                	| Description                                                	|
|-------------------------------------	|------------------------------------------------------------	|

> | installer-cr-kyma-dependencies.yaml 	| Kyma components that are required for Compass Installation 	|

Reviewer: ```suggestion
| `installer-cr-kyma-dependencies.yaml` 	| Kyma components that are required for Compass installation. 	|
```
Or is Compass Instalation a CR or another resource name here?

> | installer-cr.yaml.tpl               	| Components installed by the Compass Installer               	|

Reviewer: ```suggestion
| `installer-cr.yaml.tpl`               	| Components installed by the Compass Installer.               	|
```

## KYMA_VERSION file

> KYMA_VERSION is a file used to configure the version of Kyma used during the Compass installation.

Reviewer: ```suggestion
`KYMA_VERSION` is the file specifying the version of Kyma to use during the Compass installation.
```

### Possible values

| Value                   	| Example Value     	| Explanation             	|
|-------------------------	|-------------------	|-------------------------	|

> | `master`                	| `master`          	| Latest master version   	|
> | `master-${commit hash}` 	| `master-34edf09a` 	| Specific master version 	|
> | `PR-${PR number}`       	| `PR-1420`         	| Specific PR version     	|
> | `${Release number}`     	| `1.13.0`          	| Release version         	|

Reviewer: ```suggestion
| `master`                	| `master`          	| Latest master version.   	|
| `master-{COMMIT_HASH}` 	| `master-34edf09a` 	| Specific master version. 	|
| `PR-{PR_NUMBER}`       	| `PR-1420`         	| Specific PR version.     	|
| `{RELEASE_NUMBER}`     	| `1.13.0`          	| Release version.         	|
```

# Compass Resources

## Installation CRs

> ### [Kyma Dependencies](installer-cr-kyma-dependencies.yaml)

Reviewer 2: I think a list or a table would look better than having separate heading for each item.

Kyma components that are required for Compass Installation

### [Compass installation](.installer-cr-yaml.tpl)

> Compass Component

Reviewer 2: A bit too vague, I think.

## KYMA_VERSION file

KYMA_VERSION is a file used to configure the version of Kyma used during the Compass installation.

### Possible values

| Value                   	| Explanation             	|
|-------------------------	|-------------------------	|
| `master`                	| Latest master version   	|
| `master-${commit hash}` 	| Specific master version 	|

> | `PR-${PR number}`       	| Specific PR version     	|

Reviewer 2: What do you think about adding another column with actual example of value?

> | `${Release number`    	| Release version         	|

Reviewer 2: Missing `}`
```suggestion
| `${Release number}`    	| Release version         	|
```
